File: app/src/main/java/com/rifafauzi/moviecatalogue/model/TvShow.java
package com.rifafauzi.moviecatalogue.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class TvShow {

    @SerializedName("poster_path")
    @Expose
    private String posterPath;

    @SerializedName("overview")
    @Expose
    private String overview;

    @SerializedName("first_air_date")
    @Expose
    private String releaseDate;

    @SerializedName("id")
    @Expose
    private int id;

    @SerializedName("name")
    @Expose
    private String name;

    public TvShow(String posterPath, String overview, String releaseDate, int id, String name) {
        this.posterPath = posterPath;
        this.overview = overview;
        this.releaseDate = releaseDate;
        this.id = id;
        this.name = name;
    }

    public int getId() {
        return id;
    }

    public String getName() {
        return name;
    }

    public String getPosterPath() {
        return posterPath;
    }

    public String getOverview() {
        return overview;
    }

    public String getReleaseDate() {
        return releaseDate;
    }

}

File: app/src/main/java/com/rifafauzi/moviecatalogue/model/TvShow.kt
package com.rifafauzi.moviecatalogue.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class TvShow(@field:SerializedName("poster_path")
             @field:Expose
             val posterPath: String, @field:SerializedName("overview")
             @field:Expose
             val overview: String, @field:SerializedName("first_air_date")
             @field:Expose
             val releaseDate: String, @field:SerializedName("id")
             @field:Expose
             val id: Int, @field:SerializedName("name")
             @field:Expose
             val name: String)